modules/tensor_mechanics/test/tests/ad_linear_elasticity/applied_strain.i

```diff
@@ -0,0 +1,141 @@
+[GlobalParams]
+  displacements = 'disp_x disp_y'
+[]
+
+[Mesh]
+  type = GeneratedMesh
+  dim = 2
+  nx = 2
+  ny = 2
+  nz = 0
+  xmin = 0
+  xmax = 2
+  ymin = 0
+  ymax = 2
+  zmin = 0
+  zmax = 0
+  elem_type = QUAD4
+[]
+
+# [Modules/TensorMechanics/Master/All]
+#   strain = SMALL
+#   eigenstrain_names = eigenstrain
+#   add_variables = true
+#   generate_output = 'strain_xx strain_yy strain_xy'
+# []
+
+### Replaceable by action
+[Variables]
+  [./disp_x]
+  [../]
+  [./disp_y]
+  [../]
+[]
+
+[Kernels]
+  [./x]
+    type = ADStressDivergenceTensors
+    variable = disp_x
+    component = 0
+  [../]
+  [./y]
+    type = ADStressDivergenceTensors
+    variable = disp_y
+    component = 1
+  [../]
+[]
+
+[AuxVariables]
+  [./strain_xx]
+    order = CONSTANT
+    family = MONOMIAL
+  [../]
+  [./strain_yy]
+    order = CONSTANT
+    family = MONOMIAL
+  [../]
+  [./strain_xy]
+    order = CONSTANT
+    family = MONOMIAL
+  [../]
+[]
+
+[AuxKernels]
+  [./strain_xx_aux]
+    type = MaterialRankTwoTensorAux
+    variable = strain_xx
+    i = 0
+    j = 0
+    property = total_strain
+  [../]
+  [./strain_yy_aux]
+    type = MaterialRankTwoTensorAux
+    variable = strain_yy
+    i = 1
+    j = 1
+    property = total_strain
+  [../]
+  [./strain_xy_aux]
+    type = MaterialRankTwoTensorAux
+    variable = strain_xy
+    i = 0
+    j = 1
+    property = total_strain
+  [../]
+[]
+
+[Materials]
+  [./strain]
+    type = ADComputeSmallStrain
+    eigenstrain_names = eigenstrain
+  [../]
+[]
+### Replaceable by action
```

Comment:
What block of materials is this referring to?

Reply:
Yeah, a bit unclear. This was a marker, with all things between the two `### Replaceable by action` lines could be removed once #13098 was fixed and merged. The original non-ad test uses the action, so I wanted to duplicate as much as possible, but wanted to get this up and going in case #13098 took a long time.

Reply:
I see. Maybe in the top comment, say something like "everything between ### can be replaced by future action" or something like that